src/lib/scanner.ts

```diff
@@ -21,13 +30,15 @@ export class ScanRunner {
      */
     public async run(targets: string[]): Promise<RuleResult[]> {
         // Create the arg array.
-        const args: string[] = await this.createPathlessArgArray(targets);
+        const args: Inputs = this.createPathlessArgArray(targets);
 
         // Invoke the scanner.
-        const executionResult: ExecutionResult = await this.invokeAnalyzer(args);
```

Comment:
If we are no longer calling invokeAnalyzer... then how come I don't see it being deleted in this review?

Reply:
Deleted.